### PR TITLE
Make DataTable cells editable

### PR DESCRIPTION
**Pull request by birdsarah**
This is a re-PR of #4660 which had to get reverted after merge.

### CODE REVIEW

**Comment by birdsarah** [comment]
There were problems with the tests after merging into master.

File: bokehjs/src/coffee/models/widgets/cell_editors.coffee [hunk 1]
```diff
@@ -5,26 +5,24 @@ $2 = require "jquery-ui/spinner"
...
-
-class CellEditor extends Model
-
-class CellEditorView extends Widget.View
+class CellEditorView extends BokehView
 
   tagName: "div"
-  className: "bk-cell-editor"
+  attributes:
+    class: "bk-cell-editor"
```
Comment (mattpap) [inline bokehjs/src/coffee/models/widgets/cell_editors.coffee]: How this works? I don't see any references to `attributes` on non-Backbone.Model.

Comment (mattpap) [inline bokehjs/src/coffee/models/widgets/cell_editors.coffee]: Never mind, backbone.

**Comment by bryevdv** [comment]
@birdsarah well test are green again on the PR, do we try to merge again?

**Comment by bryevdv** [comment]
Going to merge this and see if it will pass on master.

**Comment by bryevdv** [comment]
Nope, tho now selenium shows it fails because `mget` is gone. I think we need to make PR from a @bokeh/dev contributor (so that the tests are run in the "normal" way) and cherry pick the commits to it keeping attribution so that @bgyarfas gets credit. Alternatively, we could skip the test for now. I guess it needs a merge from master in any case, to get rid of the `mget` in any case.

**Comment by bgyarfas** [comment]
@bryevdv I made you a collaborator on my branch if you wanted to make changes.

**Comment by bgyarfas** [comment]
No worries :)

**Pull request merged**